src/cartridges/int_adyen_overlay/cartridge/scripts/adyenGiving.js

```diff
@@ -54,13 +63,25 @@ function donate(donationReference, donationAmount, originalReference) {
       shopperInteraction: constants.SHOPPER_INTERACTIONS.CONT_AUTH,
     };
 
+    if (availableDonationAmounts.indexOf(parseInt(donationAmount.value)) === -1){
+      throw new Error(`Donation amount is invalid`);
+    };
+
+    if (paymentCurrency !== donationAmount.currency){
```

Comment:
looks like an issue with linting again

Reply:
It seems like linter is not checking all the files